client/src/plugins/camunda-plugin/start-instance-tool/StartInstanceTool.js

```diff
@@ -13,7 +13,7 @@ import React, { PureComponent } from 'react';
 import PlayIcon from 'icons/Play.svg';
 
 import CamundaAPI from '../shared/CamundaAPI';
-import KeyboardInteractionTrap from '../shared/KeyboardInteractionTrap';
+import KeyboardInteractionTrap from '../../../app/primitives/modal/KeyboardInteractionTrap';
```

Comment:
Can't we add the trap as a default to _all_ modals? It feels odd to me that every new modal has to add this for itself.

Reply:
I was reluctant to do that because then we need to pass `triggerAction` to the modals. Yet, we can try this out.